### PR TITLE
Local ARM64 support

### PR DESCRIPTION
**Pull request by jasonfu9966**
Updating start-dynamo and start-elasticsearch scripts to support M1 Macs. Also add a clause in readme about Airplay / API port conflict.

### CODE REVIEW

**Review by codyseibert — APPROVED** [review]

File: docs/running-locally.md [hunk 1]
```diff
@@ -84,6 +84,11 @@ All of the applications dependencies are managed via our `package.json` and `pac
 
 `npm install`
 
+### 💻 MacOS Monterey+
```
Comment (rachelschneiderman) [inline docs/running-locally.md:87]: Great update! Thanks!

**Review by rachelschneiderman — COMMENTED** [review]
I am a little concerned about modifying startup scripts for specific hardware. Has it been proved this is an issue with the machine and not an issue with a dependency like Java?

**Comment by jasonfu9966** [comment]
> I am a little concerned about modifying startup scripts for specific hardware. Has it been proved this is an issue with the machine and not an issue with a dependency like Java?

Good question! After double-checking the staging branch, I have confirmed that both dynamo and elasticsearch both fail on api load. Strangely, dynamo does not fail to start up when the start-dynamo script is started on its own, but does fail when run in context. This change fixes that.

For the elasticsearch changes, Rick and Cody both ran into issues on their M1 Macs and this change fixes that as well.

Would it help if I included a clause on the documentation about how I fixed the Java problems?

**Review by jimlerza — APPROVED** [review]

**Comment by rachelschneiderman** [comment]
> > I am a little concerned about modifying startup scripts for specific hardware. Has it been proved this is an issue with the machine and not an issue with a dependency like Java?
> 
> Good question! After double-checking the staging branch, I have confirmed that both dynamo and elasticsearch both fail on api load. Strangely, dynamo does not fail to start up when the start-dynamo script is started on its own, but does fail when run in context. This change fixes that.
> 
> For the elasticsearch changes, Rick and Cody both ran into issues on their M1 Macs and this change fixes that as well.
> 
> Would it help if I included a clause on the documentation about how I fixed the Java problems?

Got it! Thanks! I think it would be good to document whatever fixes you had to do, thank you!

**Pull request merged**